llvm/test/CodeGen/AIE/aie2/schedule/vextract.mir

```diff
@@ -114,3 +114,102 @@ body:             |
     $r3 = VEXTRACT_S16 $x3, $r16
     $r4 = VEXTRACT_S8  $x3, $r16
 ...
+
+---
+name:            vextract_to_R
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: vextract_to_R
+    ; CHECK: $r3 = VEXTRACT_S8 killed $x3, killed $r16
+    ; CHECK-NEXT: $dc0 = MOV_mv_scl killed $r4
+    $r3 = VEXTRACT_S8  $x3, $r16
+    $dc0 = MOV_mv_scl $r4
```

Comment:
Do we also want to test the no-conflict case by using instructions from different slots?